### PR TITLE
Created the banner structure and fixed the styling

### PR DESCRIPTION
**Pull request by aimunhidden485**
> First thing, PLEASE READ THIS: [ReactPlay Code Review Checklist](https://github.com/reactplay/react-play/wiki/ReactPlay-Code-Review-Checklist)

# Description
Added a dynamic banner which will change everytime the page loads

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [ ] I have performed a self-review of my own code

# Screenshots or example output

image

### CODE REVIEW

**Comment by atapas** [comment]
@aimunhidden485 The build fails with a lot of lint errors

image

Please follow the steps in readme to correct those.

**Comment by aimunhidden485** [comment]
> @aimunhidden485 The build fails with a lot of lint errors
> 
> image
> 
> Please follow the steps in readme to correct those.

okay I'm checking

File: src/common/playlists/DynamicBanner.jsx [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }, [randomPlay]);
+
+  return (
+    <Fragment>
+      <div
+        className="dynamic-banner-container"
+        style={{
+          background: `linear-gradient(rgba(0,0,0,0.5), #020808),url(${coverImage} ) center no-repeat`,
+          minHeight: '70vh'
```
Comment (atapas) [inline src/common/playlists/DynamicBanner.jsx:35]: The banner height is too large and we end up seeing only one row of play. Reduce it to 50vh

image

Also why the inline styles?

File: src/common/playlists/PlayList.jsx [hunk 2]
```diff
@@ -1,16 +1,27 @@
...
 import './playlist.css';
 import { toSanitized } from 'common/services/string';
+import DynamicBanner from './DynamicBanner';
 
 const PlayList = () => {
+  const [randomPlay, setRandomPlay] = useState({});
   const [loading, error, plays] = useGetPlays();
+  useEffect(() => {
+    const filteredPlays = plays.filter(
+      (play) => all_plays[play.component ? play.component : toSanitized(play.title_name)]
```
Comment (atapas) [inline src/common/playlists/PlayList.jsx:18]: Add a comment for this line of code.

**Review by atapas — CHANGES_REQUESTED** [review]

**Comment by atapas** [comment]
Sometime getting preview not available even of there is a cover image

image

**Comment by aimunhidden485** [comment]
> Sometime getting preview not available even of there is a cover image
> 
> image

Okay I'm checking

File: src/common/playlists/PlayList.jsx (hunk 2 above)
Comment (aimunhidden485) [reply src/common/playlists/PlayList.jsx:18]: okay

**Comment by aimunhidden485** [comment]
https://user-images.githubusercontent.com/96931784/213427398-40d04dec-bead-400d-9906-4eff9c6494be.mp4

**Review by atapas — APPROVED** [review]
Tested it with @koustov and it worked well.

**Comment by atapas** [comment]
Hi @aimunhidden485 Can we not truncate the description here?

image

Can you please fix this so that I can merge?

**Comment by aimunhidden485** [comment]
> Hi @aimunhidden485 Do not truncate the description here?
> 
> image
> 
> Can you please fix this so that I can merge?

sure

**Review by atapas — APPROVED** [review]

**Pull request merged**